system/API/ResponseTrait.php

```diff
@@ -386,32 +387,27 @@ protected function format($data = null)
 			return $data;
 		}
 
-		$config = new Format();
-		$format = "application/$this->format";
+		$mime = "application/$this->format";
 
 		// Determine correct response type through content negotiation if not explicitly declared
 		if (empty($this->format) || ! in_array($this->format, ['json', 'xml']))
 		{
-			$format = $this->request->negotiate('media', $config->supportedResponseFormats, false);
+			$config = new Format();
+			$mime = $this->request->negotiate('media', $config->supportedResponseFormats);
```

Comment:
I think we need something like:
```suggestion
			$mime = $this->request->negotiate('media', Services::format()->getConfig()->supportedResponseFormats);
```
Of course we need to add method `getConfig()` to the Format class, so we can access currently loaded config.